src/main/java/com/example/MyBatisDemo/dto/SubjectAdditionDto.java

```diff
@@ -0,0 +1,10 @@
+package com.example.MyBatisDemo.dto;
+
+import lombok.Data;
+
+@Data
+public class SubjectAdditionDto {
+    private String name;
+    private int credits;
+    private Long teacher;
```

Comment:
```suggestion
    private Long teacherId;
```
Name should be meaningful

Reply:
Done